starter/package.json

```diff
@@ -125,8 +125,8 @@
     "@angular/platform-browser-dynamic": "6.0.7",
     "@angular/platform-server": "6.0.7",
     "@angular/router": "6.0.7",
-    "@nationalbankbelgium/stark-core": "file:../dist/packages-dist/stark-core/nationalbankbelgium-stark-core-10.0.0-alpha.3-e591ed6.tgz",
-    "@nationalbankbelgium/stark-ui": "file:../dist/packages-dist/stark-ui/nationalbankbelgium-stark-ui-10.0.0-alpha.3-e591ed6.tgz",
+    "@nationalbankbelgium/stark-core": "file:../dist/packages-dist/stark-core/nationalbankbelgium-stark-core-10.0.0-alpha.3-272d4aa.tgz",
```

Comment:
Normally this kind of changes in the `package.json` should not be committed because this is changed dynamically all the time when building the packages and it causes some conflicts with other PRs in case someone else also touched this file.

Reply:
Thank you for telling me @christophercr ;)